src/helper/helper.js

```diff
@@ -310,20 +318,31 @@ function buildAbstractModelInterface_(dataLayerHelper) {
  * @private
  */
 function processCommand_(command, model) {
-  if (!isString_(command[0])) return;
+  if (!isString_(command[0])) {
+    logError(`Error processing command, no command was run. The first ` +
+        `argument must be of type string, but was of type ` +
+        `${typeof command[0]}. \nYou pushed the command\n ${command}.`,
```

Comment:
I'd avoid using "You" and similar phrases in error messages.  The reader won't always be someone familiar with the code, and in some cases using "you" can even lead to confusion.  In this case, I'd probably just say "The error occurred while processing command: ${command}"

This is true in general, but especially in web development where the browser console is just a hotkey away for most users (though hopefully no one would deploy a production site using the debug build...).

Reply:
That makes sense, writing good error messages is much trickier than I thought it would be